gammapy/background/off_data_background_maker.py

```diff
@@ -203,3 +211,59 @@ def save_models(self, modeltype):
         """
         for ngroup in range(self.ntot_group):
             self.save_model(modeltype, ngroup)
+
+    def background_links(self, data_store, modeltype, out_dir_background_model=None, outfile=None,
```

Comment:
This mostly looks OK.

I'm wondering how this is used by callers though.
Wouldn't it be simpler / more flexible to just have a method that computes the HDU index table for the background HDUs?
And maybe even put this in the `ObservationGroups` class as a method?

The fact that you're hard-coding  `'background_{}_group_{:03d}_table.fits.gz'` here, and that format is also hard-coded in `ObservationGroups` means it's hard to maintain. This should be avoided.

You could still leave the convenience method on this class if you like it, which does this
```
call ObservationGroups.make_bkg_hdu_index_table
call vstack
call write
```